### PR TITLE
Write hello to file3.

### PR DESCRIPTION
**Pull request by k7elite**
Please evaluate file3.txt
thanks.

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (Q7Elite) [inline file3.txt]: Capitalize H in hello to be Hello.

**Review by Q7Elite — CHANGES_REQUESTED** [review]
Do the the changes then we shall merge

File: file3.txt (hunk 1 above)
Comment (k7elite) [reply file3.txt]: Capitalized H in Hello
as well redistributed the code to other participants.

**Comment by k7elite** [comment]
Please review it again

**Review by Q7Elite — APPROVED** [review]
Great work.
We can go ahead with merging.

**Pull request merged**